owncloudApp/src/main/java/com/owncloud/android/presentation/authentication/LoginActivity.kt

```diff
@@ -345,53 +348,67 @@ class LoginActivity : AppCompatActivity(), SslUntrustedCertDialog.OnSslUntrusted
                 if (isSecureConnection) {
                     text = getString(R.string.auth_secure_connection)
                     setCompoundDrawablesWithIntrinsicBounds(R.drawable.ic_lock, 0, 0, 0)
+                    checkServerType(serverInfo)
                 } else {
                     text = getString(R.string.auth_connection_established)
                     setCompoundDrawablesWithIntrinsicBounds(R.drawable.ic_lock_open, 0, 0, 0)
+                    val builder = AlertDialog.Builder(context)
+                    builder.apply {
+                        setTitle(context.getString(R.string.title_dialog_insecure_http_url))
+                        setMessage(context.getString(R.string.message_dialog_insecure_http_url))
+                        setPositiveButton("Continue") { dialog, which ->
+                            checkServerType(serverInfo)
+                        }
+                        setNegativeButton("Cancel", null)
```

Comment:
"Continue" and "Cancel" strings should be also part of the string resources of the project. Otherwise, for example, a Spanish user using the app in Spanish would see these English strings, which is not what we want 👍. Check `strings.xml` because there can be already some values with those texts, or even some Android default ones (especially for "Cancel")

Reply:
Ok. I changed it! Thanks for the feedback 😃